dev/ci/gitlab.bat

```diff
@@ -43,8 +71,49 @@ IF DEFINED WIN_CERTIFICATE_PATH (
   )
 )
 
+ECHO "Finished Artifact Creation"
+TIME /T
+
+CALL :CleanupFolders
+
+ECHO "Finished Cleanup"
+TIME /T
+
 GOTO :EOF
 
+:CopyLogFiles
+  ECHO Copy log fils for artifact upload
+  MKDIR artifacts\buildlogs
+  COPY %CYGROOT%\build\buildlogs\* artifacts\buildlogs
```

Comment:
Is it not possible to move the files preserving times?

Reply:
Actually it is fine. One just can't see the timestamp when browsing files, but when downloading the zip, the timestamps are correct. Since this is not needed that frequently, I think this is fine.